.buildkite/pipeline.verify.yml

```diff
@@ -235,8 +235,8 @@ steps:
         plugins:
           - grapl-security/vault-login#v0.1.3
           - grapl-security/vault-env#v0.2.0:
-        env:
-          GRAPL_LOG_LEVEL: "DEBUG"
+        secrets:
+          - grapl/TOOLCHAIN_AUTH_TOKEN
```

Comment:
Watch the indentation here; the `secrets` object needs to be indented inside of the `vault-env` item.